tests/unit/semantic_release/version/test_algorithm.py

```diff
@@ -29,7 +29,7 @@ def test_bfs_for_latest_version_in_history():
     v1_commit = Commit(repo, binsha=b"0" * 20)
 
     class TagReferenceOverride(TagReference):
-        commit = v1_commit  # type: ignore - mocking the commit property
+        commit = v1_commit  # mocking the commit property
```

Comment:
Is mypy ok with this?

Reply:
mypy is fine with this even with the `--strict` option

Reply:
Interesting... I guess it got updated. I thought it was weird the first time I had to add it